sdk/test/UnitTests/Custom/Runtime/Credentials/InstanceProfileAWSCredentialsTest.cs

```diff
@@ -93,7 +93,6 @@ public void StaticStabilityWhenIMDSExperiencesAnOutageScenarioTest()
                 Expiration = currentTime.AddHours(6)
             };
 
-            using (new AWSConfigsDateFaker(() => currentTime.ToUniversalTime()))
```

Comment:
This line was causing the test to fail locally for me. `currentTime` is already UTC after the DateTime changes that were recently merged into `v4-development`.